ShellScript/Bash.sublime-syntax

```diff
@@ -24,6 +25,7 @@ hidden_file_extensions:
   - .bash_profile
   - .bash_variables
   - .bashrc
+  - .env
```

Comment:
I've also seen `.env.local` for environment variables which are only sourced for the local development environment. I'm not sure how common it is and whether it would be worth adding this as well?

Reply:
No real opinion here, but if we go that route, other similar candidates for consideration would include `.env.test`, `.env.ci`, `.env.dev`, `.env.development`, `.env.stg`, `.env.staging`, `.env.prod`, and `.env.production`.

I suppose something like `.env.*` does not work here?

Reply:
> I suppose something like `.env.*` does not work here?

No.